src/Nit_Compiler.lttng

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is the point of the `src/Nit_Compiler.lttng` file?

Reply:
remplace : 
lttng create "session"
lttng enable-event --userspace Nit_Compiler:Object_Instance
lttng enable-event --userspace Nit_Compiler:Object_Destroy
........

by: 
lttng load -i=~/nit/src/Nit_Compiler.lttng Nit_Compiler

This file embedded all the configuration needed in your channel as well as the buffer configuration. this also avoid the discarded events issue.